pack/tic/def_encounter.json

```diff
@@ -0,0 +1,598 @@
+		"traits": "Ocean. Island.",
+		"type_code": "location"
+	},
+	{
+		"back_flavor": "The air is frigid here, and there is no sound but the faint plinking of water dripping onto the rough stone ground. You shudder and proceed onward...",
+		"back_illustrator": "Patrick McEvoy",
+		"back_name": "Tidal Tunnel",
+		"clues": 1,
+		"code": "07174a",
+		"double_sided": true,
+		"encounter_code": "devil_reef",
+		"encounter_position": 12,
+		"faction_code": "neutral",
+		"illustrator": "Álvaro Calvo Escudero",
+		"name": "Bootlegger's Hideaway",
+		"pack_code": "def",
+		"position": 174,
+		"quantity": 1,
+		"shroud": 4,
+		"text": "<b>Forced</b> - After Bootlegger's Hideaway is revealed: Randomly choose 1 of the set-aside facedown keys and place it on Bootlegger's Hideaway without looking at it.\n[action] If you control the white key: Draw 3 cards. (Group limit once per game.)",
+		"traits": "Cave.",
+		"type_code": "location",
+		"victory": 1
+	},
+	{
+		"back_flavor": "The air is frigid here, and there is no sound but the faint plinking of water dripping onto the rough stone ground. You shudder and proceed onward...",
+		"back_illustrator": "Patrick McEvoy",
+		"back_name": "Tidal Tunnel",
+		"clues": 1,
+		"code": "07174b",
+		"double_sided": true,
+		"encounter_code": "devil_reef",
+		"encounter_position": 13,
+		"faction_code": "neutral",
+		"illustrator": "Álvaro Calvo Escudero",
+		"name": "Bootlegger's Hideaway",
+		"pack_code": "def",
+		"position": 174,
+		"quantity": 1,
+		"shroud": 4,
+		"text": "<b>Forced</b> - After Bootlegger's Hideaway is revealed: Randomly choose 1 of the set-aside facedown keys and place it on Bootlegger's Hideaway without looking at it.\n[action] If you control the black key: Gain 5 resources. (Group limit once per game.)",
+		"traits": "Cave.",
+		"type_code": "location",
+		"victory": 1
+	},
+	{
+		"back_flavor": "\"They had all kinds o' cities on the sea-bottom, an' this island was heaved up from thar.\"\n- H. P. Lovecraft, <u>The Shadow over Innsmouth</u>",
+		"back_illustrator": "Frej Agelii",
+		"back_name": "Unfathomable Depths",
+		"clues": 1,
+		"code": "07175a",
+		"double_sided": true,
+		"encounter_code": "devil_reef",
+		"encounter_position": 14,
+		"faction_code": "mythos",
+		"illustrator": "Andrew Olson",
+		"name": "Deep One Grotto",
+		"pack_code": "def",
+		"position": 175,
+		"quantity": 1,
+		"shroud": 4,
+		"text": "<b>Forced</b> - After this location is revealed: Increase its flood level. Place the set-aside purple key on it.\n[free] If investigators at this location control both the red key and the blue key and spend 2 [per_investigator] clues, as a group: Take control of the purple key and read <b>Flashback IX</b> in the Campaign Guide.",
+		"traits": "Cave. Y'ha-nthlei.",
+		"type_code": "location"
+	},
+	{
+		"back_flavor": "\"They had all kinds o' cities on the sea-bottom, an' this island was heaved up from thar.\"\n- H. P. Lovecraft, <u>The Shadow over Innsmouth</u>",
```

Comment:
Please remove the linebreak where the quotation ends (\n- will create a bullet, so just leave - and no linebreak).